flycheck.el

```diff
@@ -832,11 +832,11 @@ This variable is a normal hook.  See Info node `(elisp)Hooks'."
     (define-key map "e"         #'flycheck-set-checker-executable)
     (define-key map "?"         #'flycheck-describe-checker)
     (define-key map "h"         #'flycheck-display-error-at-point)
+    (define-key map "x"         #'flycheck-explain-error-at-point)
```

Comment:
Oh dear, I'm so sorry 😞 Thanks for the change, but I messed things up: Actually I meant "e", not "x".

I'm terribly sorry for the nuisance but could you restore the `x` binding, drop the current `e` binding, and use `e` for `explain`?  I think that's really the best binding, and `e` isn't terribly useful currently.

Really sorry to cause extra work 😞